scripts/build/package.js

```diff
@@ -76,11 +80,28 @@ data.icons.forEach(icon => {
   icon.slug = filename;
   icons.push(icon);
 
+  const iconObject = iconToObject(icon)
+
   // write the static .js file for the icon
   const jsFilepath = path.resolve(iconsDir, `${filename}.js`);
-  minifyAndWrite(jsFilepath, `module.exports=${iconToObject(icon)};`);
+  minifyAndWrite(jsFilepath, `module.exports=${iconObject};`);
+
+  // add export to the barrel file
+  const firstLetter = icon.slug[0].toUpperCase();
+  const rest = icon.slug.slice(1);
+  const iconExportName = `icon${firstLetter}${rest}`;
+
+  iconsBarrelJs.push(`${iconExportName}: ${iconObject},`);
+  iconsBarrelMjs.push(`export const ${iconExportName} = ${iconObject}`);
```

Comment:
For consistency we can consider already writing this minified (note that I also included this change in the comment above):

```suggestion
  iconsBarrelJs.push(`${iconExportName}:${iconObject},`);
  iconsBarrelMjs.push(`export const ${iconExportName}=${iconObject}`);
```